src/components/Autocomplete/Autocomlete.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { SyntheticEvent, useState } from 'react';
```

Comment:
Also, check all imports of react and add ESLint rule that makes this import redundant